Source/ACE/Entity/Landblock.cs

```diff
@@ -104,10 +104,13 @@ public Landblock(LandblockId id)
             var factoryObjects = WorldObjectFactory.CreateWorldObjects(objects);
             factoryObjects.ForEach(fo =>
             {
-                if (!worldObjects.ContainsKey(fo.Guid))
+                if (fo != null)
```

Comment:
Under what instance will WorldObjectFactory.CreateWorldObjects populate its object list with a null, and why is that not a bug in WorldObjectFactory.CreateWorldObjects?